src/TaskRunner/Commands/DrupalCommands.php

```diff
@@ -570,6 +570,35 @@ public function drupalCheckPermissions(ConsoleIO $io, array $options = [
         return $fail && $blocker ? ResultData::EXITCODE_ERROR : ResultData::EXITCODE_OK;
     }
 
+    /**
+     * Check Drupal Requirements fulfilment - from Drupal Status Page.
+     *
+     * @param array $options
+     *   Command options.
+     *
+     * @command drupal:check-requirements
+     *
+     * @option filter Filter out requirements by defined keys in config.
+     *
+     * @SuppressWarnings(PHPMD.CyclomaticComplexity)
+     * @SuppressWarnings(PHPMD.NPathComplexity)
+     */
+    public function drupalCheckRequirements(ConsoleIO $io, array $options = [
+        'filter' => false,
+    ])
+    {
+        $result = $this->taskExec($this->getBin('drush') . ' core:requirements --severity=2 --format=yaml')
```

Comment:
Filter argument can be passed to command directly to avoid parsing it later?

Reply:
Yes, we can use --ignore option, but in that case, I see need to run the command twice:

First time to get all the requirements and filter out these defined in config
Second time to get requirements defined in config by --ignoring the rest ones.
There is also this option and I'll try to find a way to filter out the results by using it and let you know: --filter[=FILTER]. Filter output based on provided expression

https://www.drush.org/12.x/commands/core_requirements/

Reply:
Hey @jonhy81 now filter argument is passed to command directly to avoid parsing it later. I hope that could do the trick.